dags/looker.py

```diff
@@ -0,0 +1,155 @@
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.models import Variable
+from airflow.providers.cncf.kubernetes.secret import Secret
+
+from operators.gcp_container_operator import GKEPodOperator
+from utils.tags import Tag
+
+DOCS = """\
+# Looker
+
+*Triage notes*
+
+As long as the most recent DAG run is successful this job can be considered healthy.
+In such case, past DAG failures can be ignored.
+
+- Failure of the `lookml_generator` task may be due to a new Glean app or changes to
+`custom-namespaces.yaml`. In these cases, the task will have created a PR in
+[looker-spoke-default](https://github.com/mozilla/looker-spoke-default)
+with the title "Auto-push from LookML Generator". These PRs will need to be merged
+and the task re-run.
+"""
+
+DEFAULT_LOOKML_GENERATOR_IMAGE_VERSION = "v1.17.0"
+
+
+default_args = {
+    "owner": "ascholtz@mozilla.com",
+    "depends_on_past": False,
+    "start_date": datetime(2024, 5, 21),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+    "retry_delay": timedelta(minutes=30),
+}
+
+tags = [Tag.ImpactTier.tier_1]
+
+looker_repos_secret_git_ssh_key_b64 = Secret(
+    deploy_type="env",
+    deploy_target="GIT_SSH_KEY_BASE64",
+    secret="airflow-gke-secrets",
+    key="probe_scraper_secret__looker_repos_secret_git_ssh_key_b64",
+)
+looker_api_client_id_prod = Secret(
+    deploy_type="env",
+    deploy_target="LOOKER_API_CLIENT_ID",
+    secret="airflow-gke-secrets",
+    key="probe_scraper_secret__looker_api_client_id_prod",
+)
+looker_api_client_secret_prod = Secret(
+    deploy_type="env",
+    deploy_target="LOOKER_API_CLIENT_SECRET",
+    secret="airflow-gke-secrets",
+    key="probe_scraper_secret__looker_api_client_secret_prod",
+)
+looker_api_client_id_staging = Secret(
+    deploy_type="env",
+    deploy_target="LOOKER_API_CLIENT_ID",
+    secret="airflow-gke-secrets",
+    key="probe_scraper_secret__looker_api_client_id_staging",
+)
+looker_api_client_secret_staging = Secret(
+    deploy_type="env",
+    deploy_target="LOOKER_API_CLIENT_SECRET",
+    secret="airflow-gke-secrets",
+    key="probe_scraper_secret__looker_api_client_secret_staging",
+)
+dataops_looker_github_secret_access_token = Secret(
+    deploy_type="env",
+    deploy_target="GITHUB_ACCESS_TOKEN",
+    secret="airflow-gke-secrets",
+    key="probe_scraper_secret__dataops_looker_github_secret_access_token",
+)
+
+
+with DAG(
+    "looker",
+    doc_md=DOCS,
+    default_args=default_args,
+    schedule_interval=None,
```

Comment:
This DAG doesn't need to be scheduled. It's either triggered from the `probe_scraper` DAG after schema deploys or via CI